components/targets/source-code-artifact/component.yaml

```diff
@@ -0,0 +1,34 @@
+name: "source-code-artifact"
+description: "Downloads archived files in zip, tag or targz format from a plain http or s3 compatible API"
+type: "target"
+parameters:
+  # Must be a valid url like: https://github.com/aws/aws-sdk-go-v2/archive/refs/heads/main.zip
+  # Or a s3 compatible url like: s3://my-bucket/my-archive.tar or gs://my-bucket/my-archive.tar.gz
+  - name: "artifact_url"
+    type: "string"
+    value: "{{ .context.trigger.event.url }}"
```

Comment:
these shouldn't be part of the open source configuration of the components, they are only relevant in the case of smithy saas